content/docs/command-reference/get.md

```diff
@@ -56,10 +56,9 @@ name.
 
 ## Options
 
-- `-o <path>`, `--out <path>` - specify a path to the desired location in the
-  workspace to place the downloaded file or directory (instead of using the
-  current working directory). Directories specified in the path will be created
-  by this command.
+- `-o <path>`, `--out <path>` - destination `path` to place the downloaded file
+  or directory (instead of using the current working directory). Directories
```

Comment:
so, does it expect file name or a directory path?

Reply:
> does it expect file name or a directory path?

Both work. You can give a dir path (ending in `/` if it doesn't yet exist) and `get/import` will create `<path>/<basename>`. Any path not ending in `/` will be the final imported data path.

Reply:
So I think "destination `path` to place the downloaded file or directory" is correct and not sure we want to make this description very long by adding all those details, which seem intuitive/expected to me.